easybuild/easyconfigs/g/GLIMPSE/GLIMPSE-2.0.0-GCC-12.2.0.eb

```diff
@@ -26,19 +26,15 @@ builddependencies = [
 ]
 
 dependencies = [
-    ('Boost', '1.81.0'),
+    ('Boost', '1.82.0'),
```

Comment:
This `Boost` version doesn't actually exist for the toolchain you've selected - were you intending to provide that too?